br-ext/board/qemu/post-build.sh

```
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+TARGETDIR="$1"
+VIRTFS_AUTOMOUNT="$2"
+VIRTFS_MOUNTPOINT="$3"
+PSS_AUTOMOUNT="$4"
```

should fail on missing or invalid data, no?